content/en/docs/instrumentation/net/getting-started/_index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Getting Started
+weight: 2
+---
+
+OpenTelemetry .NET is unique among OpenTelemetry implementations, as it is
+integrated with the .NET `System.Diagnostics` library. At a high level, you can
+think of OpenTelemetry for .NET as a bridge between the telemetry available
+through `System.Diagnostics` and the greater OpenTelemetry ecosystem, such as
+OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
+
+## Installation
+
+OpenTelemetry is available as a [NuGet package][1]. Install it with your
+preferred package manager client.
+
+## .NET Core
```

Comment:
Note that .NET Core is not the correct way to refer to it anymore. It's now:

* .NET
* .NET Framework

https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet

I'm also unsure what someone who knows they're on .NET Framework can do with this section. Should there be a link to the .NET Framework doc?

Reply:
Ah, thanks for bringing that up, ya I could include a link for .NET Framework which we already have a decent page for